charts/internal/calico/templates/custom-resource-definition/crd-bgpconfigurations.yaml

```
@@ -1,3 +1,4 @@
+{{- if semverCompare "< 1.16" .Values.global.kubernetesVersion  }}
```

You can use 
```suggestion
{{- if semverCompare "< 1.16" .Capabilities.KubeVersion.GitVersion  }}
```

(same for other occurrences